src/gate/CMakeLists.txt

```diff
@@ -12,7 +13,12 @@ add_library(Gate OBJECT
 )
 add_library(Utopia::Gate ALIAS Gate)
 
-target_include_directories(Gate PUBLIC ${PROJECT_SOURCE_DIR}/src)
+target_include_directories(Gate 
+  PUBLIC 
+    ${PROJECT_SOURCE_DIR}/src
+    ${PROJECT_SOURCE_DIR}/lib/json
+    ${PROJECT_SOURCE_DIR}/lib/kitty/include
```

Comment:
The fork does not contains EPFL libraries, so no `kitty`. Also, see the note above.